src/app/[locale]/(logged-out)/search/components/Search/Search.tsx

```diff
@@ -980,35 +947,144 @@ const Search = ({ filters, cohortDiscovery }: SearchProps) => {
                         }}
                         aria-busy={isSearching}>
                         {!isSearching && !isEuropePmcSearchNoQuery && (
-                            <Box
-                                sx={{
-                                    display: "flex",
-                                    justifyContent: "space-between",
-                                    alignItems: "center",
-                                }}>
+                            <>
                                 <Box
-                                    sx={{ display: "flex" }}
-                                    id="result-summary"
+                                    sx={{ display: "flex", paddingX: "1em" }}
```

Comment:
seem to be using `px:1` in other places - this might be the same - just check for consistency